pkg/linter/linter.go

```diff
@@ -212,22 +277,21 @@ func (lctx LinterContext) LintPackageFs(fsys fs.FS, linters []string) error {
 
 			err = linter.LinterFunc(lctx, path, d)
```

Comment:
Can LinterFunc take the fs.FS now instead of having it in the LinterContext?

(Not blocking, this lgtm)

Reply:
The linter context contains more than that, and may be needed for future linters.